src/cljs/athens/views/pages/node_page.cljs

```diff
@@ -388,13 +389,13 @@
                          :parent-uids    (set (map :block/uid (:block/parents block)))}]
     (fn [_]
       (let [{:keys [block parents embed-id]} @state
-            block (db/get-block-document (:db/id block))]
+            block (reactive/get-reactive-block-document (:db/id block))]
         [:<>
          [breadcrumbs-list {:style reference-breadcrumbs-style}
           (doall
             (for [{:keys [node/title block/string block/uid]} parents]
               [breadcrumb {:key       (str "breadcrumb-" uid)
-                           :on-click #(do (let [new-B (db/get-block-document [:block/uid uid])
+                           :on-click #(do (let [new-B (db/get-block [:block/uid uid])
```

Comment:
Fixed `do`.

This uid is not the same as in the line 392 block, it's the uid of a parent breadcrumb. When clicked, it will focus that block instead of the original block.

Reply:
Yeah, you're right